backend/src/main/java/com/bakdata/conquery/models/config/PreprocessingConfig.java

```diff
@@ -3,15 +3,13 @@
 import javax.validation.Valid;
 import javax.validation.constraints.Min;
 
-import org.hibernate.validator.constraints.NotEmpty;
-
 import lombok.Getter;
 import lombok.Setter;
 import lombok.ToString;
 
 @Getter @Setter @ToString
 public class PreprocessingConfig {
-	@NotEmpty @Valid
+	@Valid
```

Comment:
Hmm is that change intended?

Reply:
Yes, since the preprocess task has been updated to optionally accept these informations as a comand line argument. The Config is then ignored